torch/_dynamo/variables/misc.py

```diff
@@ -795,16 +796,18 @@ def call_function(
             if len(args) == 1 and args[0].has_unpack_var_sequence(tx):
                 seq = args[0].unpack_var_sequence(tx)
 
-                def func(tx, item, acc):
-                    return BuiltinVariable(sum).call_function(tx, [item, acc], {})
+                def func(tx, acc, item):
+                    return BuiltinVariable(operator.add).call_function(tx, [acc, item], {})
 
             elif (
                 len(args) == 2
                 and args[0].has_unpack_var_sequence(tx)
-                and args[1].is_callable(tx)
+                and hasattr(args[1], "call_function")
```

Comment:
No, this is not correct - if you found a case where something is not `is_callable` but doe not have a `call_function` that is a violated invariant in dynamo and should be fixed.

Reply:
So, the thing is that `is_callable` doesn't exist as a function in dynamo.

Reply:
Was it removed mistakenly?

Reply:
Or, perhaps, you meant to write (`callable(args[1])`)?

Reply:
Weird, we used to have it. In this case, you should just `call_function` without the check - that is always sound to do on a VT, and if its not there, we will graph break. If the passed in arg was not a real callable, it will break in eager correctly.

Reply:
However @voznesenskym, I wonder if keeping the check in will result in a better error message? It will say that the args are invalid, which is better than an opaque compiler message about `VariableTracker has no method 'call_function'`

Reply:
Actually you're right, we never even each this codepath since the CPython itself will throw an error about mismatched types.